python/API_Examples_Python.py

```diff
@@ -0,0 +1,200 @@
+import json     # python3 -m pip install simplejson
+import requests # python3 -m pip install requests
+import pprint   # python3 -m pip install pprintpp
+import example_sign_order
+
+
+"""
+The following is a Python Script showing the use of current API's
+The following information must be inputted before using
+user_information 
+    - traderAddr 
+    - email
+    - password
+example_sign_order.py
+- post_data
+    - traderAddr
+- Private Key
+
+"""
+testnet_url = "https://testnet271828.dex.top"
+mainnet_url = "https://dex.top"
```

Comment:
Define a variable `HOST_URL = "https://dex.top"`, and comment "For testnet use https://testnet271828.dex.top"

Reply:
done